packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -61,6 +61,8 @@ export interface TooltipProps {
   position?: 'auto' | 'top' | 'bottom' | 'left' | 'right';
   /** Tooltip trigger: click, mouseenter, focus, manual  */
   trigger?: string;
+  /** Flag to indicate that the text content is left aligned */
+  isContentLeftAligned?: boolean;
```

Comment:
Why only support a left aligned option? What if designers want to support right-aligned text, next?

Even if we don't support right-aligned text for now, something more like this would scale better in the future.

```
textPosition?: 'bottom' | 'center' | 'right';
```

Then we could create an enum

```
export enum TooltipTextPosition {
  bottom = 'bottom',
  center = 'center',
  right = 'right'
}
```

Looks like we already take this approach for the `position` property.

Reply:
Sure, I'm happy to add a right-aligned option. I was wondering why this issue was opened for left-aligned text only. @dlabrecq @tlabaj would you prefer I open a new issue to add support for right-aligned content? Or add an `enum` to this PR instead, like you have above?

Reply:
@dlabrecq @tlabaj I've opened a separate issue for this at #3089. If agreeable with you, I'm happy to assign myself to it and add support right-aligned content as well.

Reply:
I wasn't necessarily asking for right-aligned tooltips, just that we use a property that is more scalable. Once we introduce the `isContentLeftAligned` prop, it will be difficult to change without a breaking change. That's why I suggested something like the existing `position` property.

Reply:
I wasn't necessarily asking for right-aligned tooltips, just that we use a property that is more scalable. Once we introduce the isContentLeftAligned prop, it will be difficult to change without a breaking change. That's why I suggested we use something more like the existing position property.

I just want to consider a use case where we end up with both isContentLeftAligned and isContentRightAligned props together? Or, would it be better to have something more like contentPosition?: 'bottom' | 'center' | 'right' where we can easily add future support.

Reply:
Dan makes good point here.  I don't think there are plans of introducing the right aligned variant and the naming here seems to be consistent with other components.  That being said those components would be hard to change if we added another variant.

Reply:
@tlabaj Thank you for your feedback! Does that mean I should change the prop to look like `contentPosition?: 'left' | 'center'`?

Reply:
If you don't feel we'll ever have another variant, I'm fine leaving it as is. Just wanted to raise the question.

Reply:
@jenny-s51 I think we should use `contentPosition?: 'left' | 'center'`.  It's also consistent with the position enum we are already using in this component.  Can we change it to that?

Reply:
@dlabaj Absolutely -- I'm on it

Reply:
@jenny-s51 I like what @dlabrecq suggested.  But, if we are not going to have a right alignment variation I don't see the need to introduce the inconstancy.   We have at least one other component that uses isLeftAligned (or similar).
@mcarrano can you verify that there is no intent on introducing a right aligned variant.

Reply:
I see no reason to support right-aligned tooltips.